tests/integration/ipdevpoll/plugins/snmpcheck_test.py

```diff
@@ -78,5 +88,5 @@ def test_short_outage(localhost, db):
             netbox_id=localhost.pk,
             state=EventQueue.STATE_END,
         ).count()
-        == 1
+        == event_count_start + 1
```

Comment:
Ah, but here's a snag: This `EventQueue` query is actually looking for a **different** event than the first three instances. This is looking for an event with ``state=EventQueue.STATE_END``.

(A `STATE_START` event is what the plugin will post when an agent stops responding to SNMP queries. A `STATE_END` event is what it will post once responses are received again).

This assertion just happens to be true when the database is clean (i.e. starts out with 0 events), but would easily break if the initial count of start events is larger than zero.